modules/ndu/src/backend/ndu-engine.ts

```diff
@@ -449,16 +449,54 @@ export class UnderstandingEngine {
             activeWorkflow: tn.activeWorkflow,
             nodeId: tn.name
           })
-        } else if ((<sdk.ListenNode>node)?.triggers?.length) {
+        } else if ((<sdk.ListenNode>node)?.triggers?.length || node.type === 'prompt') {
           const ln = node as sdk.ListenNode
+
+          if (node.type === 'prompt') {
+            // TODO: Add triggers on the node itself instead of hardcoded here
+            ln.triggers = [
+              {
+                name: 'prompt_yes',
+                effect: 'prompt.inform',
+                conditions: [{ id: 'prompt_listening' }, { id: 'user_intent_yes' }]
+              },
+              {
+                name: 'prompt_no',
+                effect: 'prompt.inform',
+                conditions: [{ id: 'prompt_listening' }, { id: 'user_intent_no' }]
+              },
+              {
+                name: 'prompt_inform',
+                effect: 'prompt.inform',
+                conditions: [
+                  { id: 'prompt_listening' },
+                  { id: 'custom_confidence', params: { confidence: 0.7 } } // TODO: inform by type of prompt
+                  // { id: 'user_intent_is', params: { intentName: 'inform' } } // TODO: potentially custom intent
```

Comment:
This whole block is a TODO, because triggers should be moved directly to the prompt node when created or modified on the studio. But since it's still changing, we keep it there for now